### PR TITLE
Release dcos monitoring 1.4.0

### PR DESCRIPTION
**Pull request by janisz**
## Jira Ticket
Before creating this pull request, make sure you have a separate ticket for the docs team to track their work. If you need assistance, ask in the #documentation Slack channel.

https://jira.d2iq.com/browse/D2IQ-71300

## Description of changes being made

Release of 1.3.1
Release of 1.4.0

## Checklist
- [x] Change all affected versions, if applicable (e.g. 1.13, 2.0, 2.1).
- [ ] Test all commands and procedures, if applicable.
- [x] Create your PR against `staging`, not `master`. 
- [x] Update all links if you are moving a page.
- [x] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> ie `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://wiki.d2iq.com/display/ENG/Contributing+to+Docs) for more information.

### CODE REVIEW

File: .github/styles/ttd-light/So.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 extends: existence
 message: "Don't start a sentence with '%s'"
-ignorecase: true
+ignorecase: false
```
Comment (cmastr) [inline .github/styles/ttd-light/So.yml:4]: @janisz what is this change doing?

Comment (janisz) [reply .github/styles/ttd-light/So.yml:4]: the regexp `'(?:[;-]\s)so[\s,]|So[\s,]'` is case sensitive. If we ignore case we will have false positives 
see https://regexr.com/5b237

File: pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/alertmanager/config/index.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+The Git repository can be either public or private.
+You can omit the `credentials` section if the Git repository is public.
+If the Git repository is private, you must configure the credentials to access the Git repository.
+
+## Create secrets for Git repository credentials
+
+If the Git repository containing the Alertmanager configurations is private, you must configure the secrets first.
+Currently, the following Auth types are supported.
+
+<p class="message--important"><strong>IMPORTANT: </strong>If the Git repository contains submodules, all the submodules must use the same Auth type as what's used for the repository. For example, if HTTP Auth is configured for the Git repository, all of the submodules in the repository's <code>.gitmodules</code> file must use the <code>https</code> URL scheme.</p>
```
Comment (cneth) [inline pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/alertmanager/config/index.md:46]: I see we changed "what is" to "what's" here but I don't think it improved this sentence very much.

I suggest removed "as what's" altogether -->

"If the Git repository contains submodules, all the submodules must use the same Auth type as the one used for the repository"

This same warning appears in several pages, let's make sure we fix all of them, I didn't add comments for all of the incidences.

File: pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/alertmanager/config/index.md [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+}
+```
+
+### SSH Auth
+
+```bash
+dcos security secrets create -f <PATH_TO_PRIVATE_KEY> gitsshkey-secret
+```
+
+For GitHub, you must add the [Deployment Key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) (that's, the public key) to the repository.
```
Comment (cneth) [inline pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/alertmanager/config/index.md:80]: The change from "that is" -> "that's" here, while probably technically correct, is awkward and we can see Vale also complains.

Please change back to "(that is, the public)", or just "(the public key)

File: pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/alertmanager/slack/index.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+model: ../../../data.yml
+---
+
+# Configuring Alertmanager to alert via Slack
+
+## Create an Alertmanager configuration file
+
+See the [Alertmanager configuration documentation](https://prometheus.io/docs/alerting/configuration/) and the [Slack configuration section](https://prometheus.io/docs/alerting/configuration/#slack_config) for a detailed overview of how to create a configuration file with a Slack receiver.
+
+See the following example of a basic configuration file that's set up with a Slack receiver.
```
Comment (cneth) [inline pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/alertmanager/slack/index.md:17]: "that's" should be avoided.

Suggest we reword this as:

"Below is an example of a simple configuration file with a Slack receiver"

File: pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/alertmanager/slack/index.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+It's possible to trigger a reload of the Alertmanager configurations after the service is installed.
+
+```bash
+dcos monitoring plan start reload-alertmanager-config
+```
+
+## Testing Slack alerts
+
+You can test that Alertmanager is configured correctly by SSH to the node on which Alertmanager is running and manually hitting the Alertmanager endpoint to trigger an alert.
```
Comment (cneth) [inline pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/alertmanager/slack/index.md:118]: "correctly by SSH to the node" is awkward.

how about:

"configured correctly by using SSH to connect to the node"

File: pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/alertmanager/storage/index.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+
+You can configure Alertmanager to use a dedicated volume provided by the [DC/OS Storage Service](/services/storage/latest/) (DSS).
+
+At least one volume must be available for Alertmanager to use.
+For more information on creating volumes, see the [DC/OS Storage Service](/services/storage/latest/cli-references/dcos-storage-volume/dcos-storage-volume-create) documentation.
+
+When configuring Alertmanager, select `MOUNT` for its volume `type`, choose the minimum `size` for the volume in MB, and enter the `profile` for the volume.
+For more information on volume profiles, see the [DC/OS Storage Service](/services/storage/latest/cli-references/dcos-storage-profile/) documentation.
+
+The following configuration will deploy Alertmanager to a volume with a profile of `fast` that's 512 MB or larger.
```
Comment (cneth) [inline pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/alertmanager/storage/index.md:23]: "that's" -> "that is"

File: pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/grafana/storage/index.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+
+You can configure Grafana to use a dedicated volume provided by the [DC/OS Storage Service](/services/storage/latest/) (DSS).
+
+At least one volume must be available for Grafana to use.
+For more information on creating volumes, see the [DC/OS Storage Service](/services/storage/latest/cli-references/dcos-storage-volume/dcos-storage-volume-create) documentation.
+
+When configuring Grafana, select `MOUNT` for its volume `type`, choose the minimum `size` for the volume in MB, and enter the `profile` for the volume.
+For more information on volume profiles, see the [DC/OS Storage Service](/services/storage/latest/cli-references/dcos-storage-profile/) documentation.
+
+The following configuration will deploy Grafana to a volume with a profile of `fast` that's 512 MB or larger.
```
Comment (cneth) [inline pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/grafana/storage/index.md:23]: "that's" --> "that is"

File: pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/grafana/dashboard-configs/index.md [hunk 8]
```diff
@@ -0,0 +1,154 @@
...
+}
+```
+
+### SSH Auth
+
+```bash
+dcos security secrets create -f <PATH_TO_PRIVATE_KEY> gitsshkey-secret
+```
+
+For GitHub, you must add the [Deployment Key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) (that's the public key) to the repository.
```
Comment (cneth) [inline pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/grafana/dashboard-configs/index.md:98]: The change from "that is" -> "that's" here, while probably technically correct, is awkward and we can see Vale also complains.

Please change back to "(that is, the public)", or just "(the public key)

File: pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/prometheus/alert-rules/index.md [hunk 9]
```diff
@@ -0,0 +1,167 @@
...
+}
+```
+
+### SSH Auth
+
+```bash
+dcos security secrets create -f <PATH_TO_PRIVATE_KEY> gitsshkey-secret
+```
+
+For GitHub, you must add the [Deployment Key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) (that's the public key) to the repository.
```
Comment (cneth) [inline pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/operations/prometheus/alert-rules/index.md:111]: The change from "that is" -> "that's" here, while probably technically correct, is awkward and we can see Vale also complains.

Please change back to "(that is, the public)", or just "(the public key)

File: pages/mesosphere/dcos/services/dcos-monitoring/1.4.0/operations/alertmanager/config/index.md [hunk 10]
```diff
@@ -0,0 +1,158 @@
...
+}
+```
+
+### SSH Auth
+
+```bash
+dcos security secrets create -f <PATH_TO_PRIVATE_KEY> gitsshkey-secret
+```
+
+For GitHub, you must add the [Deployment Key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) (that's, the public key) to the repository.
```
Comment (cneth) [inline pages/mesosphere/dcos/services/dcos-monitoring/1.4.0/operations/alertmanager/config/index.md:80]: The change from "that is" -> "that's" here, while probably technically correct, is awkward and we can see Vale also complains.

Please change back to "(that is, the public)", or just "(the public key)

File: pages/mesosphere/dcos/services/dcos-monitoring/1.4.0/operations/grafana/dashboard-configs/index.md [hunk 11]
```diff
@@ -0,0 +1,154 @@
...
+}
+```
+
+### SSH Auth
+
+```bash
+dcos security secrets create -f <PATH_TO_PRIVATE_KEY> gitsshkey-secret
+```
+
+For GitHub, you must add the [Deployment Key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) (that's the public key) to the repository.
```
Comment (cneth) [inline pages/mesosphere/dcos/services/dcos-monitoring/1.4.0/operations/grafana/dashboard-configs/index.md:98]: The change from "that is" -> "that's" here, while probably technically correct, is awkward and we can see Vale also complains.

Please change back to "(that is, the public)", or just "(the public key)

File: pages/mesosphere/dcos/services/dcos-monitoring/1.4.0/operations/grafana/dashboard-configs/index.md [hunk 12]
```diff
@@ -0,0 +1,154 @@
...
+    }
+  }
+}
+```
+
+<p class="message--note"><strong>NOTE: </strong>You will have to use `git@github.com:<USER>/<REPO>.git` instead of <code>https</code> as the scheme of the URL.</p>
+
+## Fetching from a branch in a Git repository
+
+By default, the service will fetch from the master branch (that's `refs/heads/master`) of the Git repository.
```
Comment (cneth) [inline pages/mesosphere/dcos/services/dcos-monitoring/1.4.0/operations/grafana/dashboard-configs/index.md:120]: "that's" -> "that is"

File: pages/mesosphere/dcos/services/dcos-monitoring/1.4.0/operations/grafana/storage/index.md [hunk 13]
```diff
@@ -0,0 +1,38 @@
...
+
+You can configure Grafana to use a dedicated volume provided by the [DC/OS Storage Service](/services/storage/latest/) (DSS).
+
+At least one volume must be available for Grafana to use.
+For more information on creating volumes, see the [DC/OS Storage Service](/services/storage/latest/cli-references/dcos-storage-volume/dcos-storage-volume-create) documentation.
+
+When configuring Grafana, select `MOUNT` for its volume `type`, choose the minimum `size` for the volume in MB, and enter the `profile` for the volume.
+For more information on volume profiles, see the [DC/OS Storage Service](/services/storage/latest/cli-references/dcos-storage-profile/) documentation.
+
+The following configuration will deploy Grafana to a volume with a profile of `fast` that's 512 MB or larger.
```
Comment (cneth) [inline pages/mesosphere/dcos/services/dcos-monitoring/1.4.0/operations/grafana/storage/index.md:23]: "that's" -> "that is"

File: .github/styles/ttd-light/So.yml (hunk 1 above)
Comment (cmastr) [reply .github/styles/ttd-light/So.yml:4]: ok gotcha.

File: pages/mesosphere/dcos/services/dcos-monitoring/1.4.0/operations/grafana/storage/index.md (hunk 13 above)
Comment (janisz) [reply pages/mesosphere/dcos/services/dcos-monitoring/1.4.0/operations/grafana/storage/index.md:23]: No, I changed all of this because linter told me to do so.

@cmastr Please fix linter!

**Comment by janisz** [comment]
@cneth Thanks for review. I'll fix linter to not complain about `that is` →`that's` and other abbreviation.

File: pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/cli-references/monitoring-prometheus/monitoring-prometheus-query/index.md [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+navigationTitle:  dcos monitoring prometheus query
+title: dcos monitoring prometheus query
+menuWeight: 1
+excerpt: Evaluate an instant query at a single point in time
+enterprise: false
+---
+
+# Description
+
+The `dcos monitoring prometheus query` command evaluates an instant query at a given time.
```
[the remark this answers is not in the document]
Comment (janisz) [reply pages/mesosphere/dcos/services/dcos-monitoring/1.3.1/cli-references/monitoring-prometheus/monitoring-prometheus-query/index.md:12]: It was linter suggestion to not use `a single point in time`

**Comment by janisz** [comment]
@cmastr Can you look at this?

**Review by cmastr — APPROVED** [review]
